psi4/src/psi4/lib3index/dfhelper.cc

```diff
@@ -324,28 +324,26 @@ void DFHelper::prepare_sparsity() {
         }
     }
 
-    // get screening tolerance
+    // => Prepare screening/indexing data <=
     double tolerance = cutoff_ * cutoff_ / max_val;
 
-    //#pragma omp parallel for simd num_threads(nthreads_) schedule(static)
-    for (size_t i = 0; i < pshells_ * pshells_; i++) schwarz_shell_mask_[i] = (shell_max_vals[i] < tolerance ? 0 : 1);
+    // ==> Is this shell pair significant? <==
+    for (size_t i = 0; i < pshells_ * pshells_; i++) schwarz_shell_mask_[i] = (shell_max_vals[i] >= tolerance);
 
-    //#pragma omp parallel for private(count) num_threads(nthreads_)
+    // ==> Is this basis function pair significant? Also start storing non-symmetric indexing. <==
```

Comment:
why remove a starting-point for future parallelization work?

Reply:
I assumed the line didn't work, if it was commented out. Do you know/suspect otherwise?

Reply:
I expect you're right that it doesn't work for some reason. But Matt was proficient in threading, so if he added it, it might be a good starting-point.

Reply:
I would rather have the line removed. Do either of you care enough to make this a barrier to merging this in? I have no interest in mediating a disagreement between the two of you.

Reply:
Zach judged the parallelism loop on its merits, which I didn't. Removal fine by me, then. :-)